.buildkite/scripts/upload-release.sh

```diff
@@ -206,6 +221,7 @@ function create_release() {
 
   update_github_release "$tag"
   create_sentry_release "$tag"
+  send_bench_webhook "$tag"
```

Comment:
@Electroid this is untested and I'm assuming things here, let me know if its wrongly done or if better way.

also this needs the `BENCH_URL` to make it more hidden from public, the domain/ip is mentioned in the discord.